lib/maze/bitbar_devices.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Maze
+  # Provides a source of capabilities used to run tests against specific BrowserStack devices
+  # noinspection RubyStringKeysInHashInspection
+  class BitBarDevices
+    APPIUM_1_9_1 = '1.9.1'
+    APPIUM_1_15_0 = '1.15.0'
+    APPIUM_1_20_2 = '1.20.2'
```

Comment:
Looks like these are unused?

Reply:
This is configurable for Android, so will probably come up with the next step of running Android apps